sample-viewmodel/src/main/java/com/example/molecule/viewmodel/presentationLogic.kt

```diff
@@ -38,12 +38,12 @@ data class Model(
 class PupperPicsViewModel : MoleculeViewModel<Event, Model>() {
   @Composable
   override fun models(events: Flow<Event>): Model {
-    return PupperPicsPresenter(events, PupperPicsService())
+    return pupperPicsPresenter(events, PupperPicsService())
   }
 }
 
 @Composable
-fun PupperPicsPresenter(events: Flow<Event>, service: PupperPicsService): Model {
+fun pupperPicsPresenter(events: Flow<Event>, service: PupperPicsService): Model {
```

Comment:
Should we change this naming convention with the casing change?

Reply:
Seems fine to me. @chris-horner?

Reply:
🤷 

I usually use lowercase when decomposing presenters in my own work. So: feels fine

Reply:
All g with me 👍